README.md

```diff
@@ -1,7 +1,8 @@
-# features
+# Feature Tracking and Backlog
+
 Feature tracking repo for Kubernetes releases
 
-This repo only contains issues.  These issues are umbrellas for new features added to Kubernetes.  
+This repo only contains issues. These issues are umbrellas for new features to be added to Kubernetes. A feature can take multiple releases to complete. And a feature can be tracked as backlog items before work begins.
 
```

Comment:
For better or worse, you should consider a feature anything that:

- you would write a blog post about; and/or
- anything you need to talk to more than one person about implementing (beyond your SIG).

We are not going to develop a universal definition that will satisfy all "features" but we feel that this is a good starting point. There will always be exceptions. That said, when in doubt, **create a feature proposal.**